app/components/publications/publication-sidebar.js

```diff
@@ -79,25 +80,19 @@ export default class PublicationsPublicationSidebarComponent extends Component {
   @action
   setRegulationType(regulationType) {
     this.publicationFlow.regulationType = regulationType;
-    if (this.args.didChange) {
-      this.args.didChange(this.publicationFlow, 'regulationType');
-    }
+    this._notify(this.publicationFlow, ['regulationType']);
   }
 
   @action
   setPublicationMode(publicationMode) {
     this.publicationFlow.mode = publicationMode;
```

Comment:
Dit werd vroeger gedaan met
```
const selectedPubMode = this.publicationModes.find(...);
this.publicationFlow.mode = selectedPubMode;
```
Dit zorgde ervoor dat de dropdown het juiste model selecteerde (om een of andere reden maakt ember-data 2 publication modes).